
#!/usr/bin/env python
PACKAGE = "add_pallet"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pallet_type_param", str_t, 0, "Name of new pallet type", "NULL")
gen.add("width_param", int_t, 0, "width of pallet (meters)", 100, 0, 200)
gen.add("depth_param", int_t, 0, "depth of pallet (meters)", 100, 0, 200)

exit(gen.generate(PACKAGE, "add_pallet", "dynamic_params"))